frontend/src/hooks/useFields.tsx

```diff
@@ -82,11 +111,21 @@ export function useFields<T extends Zaak = Zaak>(
       name: "zaaktype",
       filterLookup: "zaaktype__in",
       filterValue: searchParams.get("zaaktype__in") || "",
-      valueTransform: (v) =>
-        zaaktypeChoices.find((c) => c.value === v.zaaktype)?.label || (
-          <Placeholder />
-        ),
-      options: zaaktypeChoices,
+      valueTransform: ({ zaaktype }) => {
+        // Zaaktype choices not yet loaded.
+        if (zaaktypeChoices === null) {
+          return <Placeholder />;
+        }
+
+        // Find label by zaaktype choice.
+        const zaaktypeChoice = zaaktypeChoices.find(
+          ({ value }) => value === zaaktype,
+        );
+
+        // Return label, or null.
+        return zaaktypeChoice?.label || zaaktype;
+      },
```

Comment:
I am confused :thinking: This filter should only be `zaaktype` now, it was changed here: https://github.com/maykinmedia/open-archiefbeheer/blob/a4cefce349622aac169599f00501a03403b9eb88/frontend/src/hooks/useFields.tsx#L87

Reply:
Ah! I see the conflicts now :smile: